src/components/TextField/TextField.css

```diff
@@ -0,0 +1,69 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+@import "src/css/variables/colors.css";
+@import "src/css/variables/fonts.css";
+
+.y-text-field .ms-Label {
```

Comment:
should probably add a comment saying this is temporary tech debt until we can set styling/theme options for all Fabric label instances (that work is in progress)